### PR TITLE
Add blog post about Java 17

### PR DESCRIPTION
**Pull request by colebow**
Quick, easy blog post announcing the update to Java 17.

### CODE REVIEW

**Comment by martint** [comment]
Just to clarify, we've only updated the runtime requirement. We're still targeting Java 11 APIs and bytecode and will keep doing that for a few weeks in case something unexpected comes up and we need to revert.

File: _posts/2022-07-14-trino-updates-to-java-17.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+You've already read the title, and it's exciting news - as of Trino version 390,
+which releases today, Trino has officially been updated from Java 11 to Java 17.
+This comes with a few changes, the most important of which is that if you've
+been running Trino on Java 16 or older, you'll need to update Java to utilize
+Trino versions 390 and later. It's also worth mentioning that newer versions of
+Java, such as Java 18 or 19, are not supported - they might work, but they
+haven't been tested or benchmarked - Java 17 is the new, recommended version for
+Trino.
+
+excerpt_separator: <!--more-->
```
Comment (mosabua) [inline _posts/2022-07-14-trino-updates-to-java-17.md:17]:
```suggestion
<!--more-->
```

File: _posts/2022-07-14-trino-updates-to-java-17.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+The reason this change is exciting is that using a new and better version of
+Java will allow us to better improve Trino, too! With new language features,
```
Comment (mosabua) [inline _posts/2022-07-14-trino-updates-to-java-17.md:20]: better improve..  .. reword

File: _posts/2022-07-14-trino-updates-to-java-17.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+there are more tools in the development toolkit, and with the update leading to
```
Comment (mosabua) [inline _posts/2022-07-14-trino-updates-to-java-17.md:21]: Separate this whole paragraph into two aspects

* new runtime requirement now is 17 and that has x advantages
* in one of the upcoming release we will switch build/code level requriement to 17 .. and that allows us to make the code cleaner and better

File: _posts/2022-07-14-trino-updates-to-java-17.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+the Java language performing slightly better on the whole, you may see some
+small, across-the-board performance improvements when switching from Java 11 to
+Java 17. When you've got the time, we strongly recommend making the upgrade!
```
Comment (mosabua) [inline _posts/2022-07-14-trino-updates-to-java-17.md:24]: add conclusion section that we are excited and that you can learn more in 


https://trino.io/episodes/36.html

and we also mentioned this in 

https://trino.io/episodes/35.html

and have planned and worked on this with link to JIRA issue

And give a shout out to @wendigo for all the hard work

File: _posts/2022-07-14-trino-updates-to-java-17.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title:  "Trino Updates to Java 17"
```
Comment (mosabua) [inline _posts/2022-07-14-trino-updates-to-java-17.md:3]:
```suggestion
title:  "Trino updates to Java 17"
```
```suggestion
title:  "Trino Updates to Java 17"
```

File: _posts/2022-07-14-trino-updates-to-java-17.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+The plan is to update the build to Java 17 a few weeks from now, which will also
+allow us to use Java 17 APIs and the changes to the language in Trino code. With
+new language features, there are more tools in the development toolkit, and
+it'll allow us to write cleaner and better code moving forwards.
+
+This upgrade has been in the works for a while and been a long time coming, so
+if you want to learn more about the specifics, one of the best places to check
+that out is the Trino Community Broadcast. Updating to Java 17 was the focus of
+[Episode 36](https://trino.io/episodes/36.html), and we also talked about it
```
Comment (mosabua) [inline _posts/2022-07-14-trino-updates-to-java-17.md:35]: episode 36

File: _posts/2022-07-14-trino-updates-to-java-17.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+previously in [Episode 35](https://trino.io/episodes/35.html). If you want to
```
Comment (mosabua) [inline _posts/2022-07-14-trino-updates-to-java-17.md:36]: episode 35

**Comment by mosabua** [comment]
Any other input @wendigo !

**Comment by electrum** [comment]
We might mention that if you're running the Docker image, then you don't need to do anything, since it automatically includes the correct version of Java.

**Comment by wendigo** [comment]
🎉

**Pull request merged**